lib/fetch/body.js

```diff
@@ -7,18 +7,21 @@ const { FormData } = require('./formdata')
 const { kState } = require('./symbols')
 const { webidl } = require('./webidl')
 const { DOMException, structuredClone } = require('./constants')
-const { Blob } = require('buffer')
+const { Blob, File: NativeFile } = require('buffer')
 const { kBodyUsed } = require('../core/symbols')
 const assert = require('assert')
 const { isErrored } = require('../core/util')
 const { isUint8Array, isArrayBuffer } = require('util/types')
-const { File } = require('./file')
+const { File: UndiciFile } = require('./file')
```

Comment:
Can we just change the `./file` and dynamically change the module.exports if NativeFile exists?

Reply:
The issue with that is that undici's File class won't be accessible in node 19.x and above.

ie. you can import the File class by doing
```js
import { File } from 'undici'
```